articles/libraries/lock/v10/lock-passwordless-migration-guide.md

```diff
@@ -0,0 +1,138 @@
+---
+section: libraries
+toc: true
+description: lock-passwordless to Lock 10 with Passwordless Mode Migration Guide
+---
+# lock-passwordless to Lock 10 with Passwordless Mode Migration Guide
+
+The following instructions assume you are migrating from **lock-passwordless** to the latest **Lock 10** using the **Passwordless** mode.
+
+The goal of this migration guide is to provide you with all of the information you would need to update your lock-passwordless installation to Lock 10 using the Passwordless Mode. Your first step is to remove lock-passwordless from your project and include the latest version of Lock 10. Beyond that, take a careful look at each of the areas on this page. You will need to change your implementation to reflect the new changes, not only the initialization of `Auth0LockPasswordless` and your calls to Lock methods, but especially any customization options you were implementing may need inspected and changed. Take a look below for more information!
+
+## General Changes and Additions
+
+### Importing Auth0LockPasswordless
+If you're loading from the CDN, you can still use `Auth0LockPasswordless`. The difference is that you'll provide options in the constructor, like we do with `Auth0Lock`:
```

Comment:
Make sure there are empty lines after headings so the linter will be happy :D (there are a few of these)